### ISSUE TITLE
Reorganization of the plot of land interface, in particular for waste garbage cans

### ISSUE THREAD

**Issue by LPoin** [georchestra/mapstore2-cadastrapp#198]
A request has been made to modify the “dustbin” type icons in the “Parcel search” interface.
The interface must be modified to change the location of certain buttons. 
The change of location will allow a better understanding of the link between the buttons and other interface elements 
(e.g. changing the location of the “+” to add a tab).

- Add a tab :
Adding a “+” tab to add a selection tab and removing the “+” in the toolbar to the right of the tabs.
Clicking on the + will add a tab at the end with the same behavior as the current “+” button.

Technically, a NavItem tab will be added after all the other tabs :
https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/components/PlotSelection.jsx#L116

Click on "+" add a tab with the same behaviour as the current "+" button :
https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/components/PlotSelection.jsx#L66

- Remove a tab : 
Adding a cross on all tabs to delete a tab, and removing the trash can to the right of the tabs (currently next to the “+”).

For this purpose, we will add a cross-shaped button to the right of the title
https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/components/PlotSelection.jsx#L103-L105

Clicking on the cross will execute the action of deleting a tab with the index number to be deleted.
https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/components/PlotSelection.jsx#L74

- Delete a search : 
The location of search-related buttons is not optimal. These actions should be at the bottom of the “parcel search” zone and not to the right of the “parcel selection” heading.

![image](https://github.com/user-attachments/assets/e4512eca-f6ab-4a99-a557-67fa4f70ec2d)

- CSS : 
The CSS styling of the active and inactive buttons is not optimal, with borders that appear to be poorly displayed when the button is active.

**Comment by MarieSuet** [comment]
After discussion with @landryb, a small remark about the loupe icon:  we're afraid that users will be lost if the option to search for a parcel by reference, cadastral address or identifier is displayed when they click on the manual selection icons (point, line, polygon): wouldn't it be better to display “Click on the map to select one or more parcels” instead ? And if they want to search for the parcel ID, display this option ONLY when they click on the small loupe ? @jusabatier @fvanderbiest 
Tell me if unclear

**Comment by jusabatier** [comment]
Maybe labelling with "Sur la carte :" before map selection buttons ?

For me we shouldn't have a loupe icon that hide search options when not selected. 

It will make the functionality more complex to use as some user may not understand that they have to click on it to display the search form.

**Comment by landryb** [comment]
> It will make the functionality more complex to use as some user may not understand that they have to click on it to display the search form.

so it's not confusing for your users to have the point tool selected/highlighted, and the search fields displayed at the same time ? To me here we have two distinct search entries here:
- by clicking on the map (point, line, polygon, uf)
- by fields/attributes

and i'm not sure they should be displayed 'at the same time'.. but maybe that's fine for end-users

**Comment by Gaetanbrl** [comment]
> and i'm not sure they should be displayed 'at the same time'.. but maybe that's fine for end-users

This is a point to discuss. Currently, cadastrapp allow to display `"search by field"` and `"select by click" `at the same time (the native proposal respect this capacity) : 

> See rennes métropole current version (portail-test)
![image](https://github.com/user-attachments/assets/7540e3c1-e0e3-4c3d-b254-0e4e413fbfed)

**Comment by jusabatier** [comment]
> so it's not confusing for your users to have the point tool selected/highlighted, and the search fields displayed at the same time ?

No, I think they will be more confused to not find the search field and may not understand/forget that they have to click on a magnifier icon to show them.

For me, display both map and references search tools is not a problem as long as it remains easy to use and understand.

**Comment by MarieSuet** [comment]
> Maybe labelling with "Sur la carte :" before map selection buttons ?

I agree, if we're going to keep the search possible even when we're on the point/line/polygon icons, at least put something to indicate that you have to click on the map to select parcels. Otherwise, I'm afraid we'll lose clarity...

**Comment by LPoin** [comment]
In PR for review [here](https://github.com/georchestra/mapstore2-cadastrapp/pull/207)

**Comment by pierrejego** [comment]
Closed with #207
